### ISSUE TITLE
each spl_autoload_functions() can instanceof \Closure at Kahlan\Jit\ClassLoader line 114

### ISSUE THREAD

**Issue by samsonasik** [kahlan/kahlan#339]
When the data of `spl_autoload_functions()` instanceof Closure, we will get error: 

```bash
Fatal error: Uncaught Error: Cannot use object of type Closure as array in /Users/samsonasik/www/app/vendor/kahlan/kahlan/src/Jit/ClassLoader.php on line 114
```

**Comment by jails** [comment]
Not sure to get the issue, would you mind creating a small project example with the issue ?

**Comment by samsonasik** [comment]
@jails I created reproducible repo at https://github.com/samsonasik/kahlan-issue-339 .

**Comment by jails** [comment]
Thank you :+1, fixed in 4.7.2
